src/utils.ts

```diff
@@ -529,6 +541,20 @@ export class Utils {
     (target || e.target).dispatchEvent(simulatedEvent);
   }
 
+  public static getScaleForElement(element: HTMLElement) {
+    let elementToGetTheScaleFrom = element;

```

Comment:
why re-assing ? you never use 'element' again. just shorten to el and use it below...